docs/resources/Guild.md

```diff
@@ -497,6 +498,64 @@ Giving the member a role will bypass Membership Screening as well as the guild's
 > warn
 > We are making significant changes to the Membership Screening API specifically related to getting and editing the Membership Screening object. Long story short is that it can be improved. As such, we have removed those documentation. There will **not be** any changes to how pending members work, as outlined above. That behavior will stay the same.
 
+### AutoMod Rule Object
+
+###### AutoMod Rule Structure
+
+| Field            | Type                | Description                                                                                            |
+| ---------------- | ------------------- | ------------------------------------------------------------------------------------------------------ |
+| name             | string              | the auto-mod rule name                                                                                 |
```

Comment:
```suggestion
| name             | string              | the rule name                                                                                 |
```
i think you should take all these "auto-mod" from the rows, its title is already explaining that it's a automod rule

Reply:
Done!